torch/utils/data/dataloader.py

```diff
@@ -212,7 +229,8 @@ def __init__(self, dataset, batch_size=1, shuffle=False, sampler=None,
                 sampler = _InfiniteConstantSampler()
             else:  # map-style
                 if shuffle:
-                    sampler = RandomSampler(dataset, generator=generator)
+                    # Cannot statically verify that dataset is Sized
+                    sampler = RandomSampler(dataset, generator=generator)  # type: ignore
```

Comment:
Just checking -- this is a pre-existing problem, right? I see that RandomSampler before this PR takes `Sized`, and Dataset before this PR is a `Generic[T_co]`

Reply:
That's correct